code/modules/modular_computers/computers/modular_computer/interaction.dm

```diff
@@ -191,10 +191,13 @@
 
 // On-click handling. Turns on the computer if it's off and opens the GUI.
 /obj/item/modular_computer/attack_self(var/mob/user)
-	if(enabled && screen_on)
-		ui_interact(user)
-	else if(!enabled && screen_on)
-		turn_on(user)
+	if(user.is_necromorph())
```

Comment:
this doesn't belong here for lots of reasons

Reply:
it does because attacking modular computer redirects you here. Anyway I still need to replace this check with advanced tool user (or add canusetopic if that won't break anything)